TFT/src/User/API/Temperature.c

```diff
@@ -68,7 +82,9 @@ void heatSetCurrentTemp(uint8_t index, int16_t temp)
 // Get current temperature
 int16_t heatGetCurrentTemp(uint8_t index)
 {
-  index = fixHeaterIndex(index);
+  if (heaterIndexInvalid(index))
+    return 0;
+
```

Comment:
The Extruder menu needs temperature status for a specific extruder instead of hotend. for example, a dual hotend setup will have the same hotend index as the extruders but a mixing hotend with 3 extruders will have the same hotend index (`NOZZLE0`) for all extruders.
With the new logic, the hotend temperature for all extruders except `E0` for mixing hotend setup will always return 0. This will trigger false cold extrusion warnings for all extruders except `E0`.